fhir-bulkimportexport/src/main/java/com/ibm/watson/health/fhir/bulkexport/ChunkWriter.java

```diff
@@ -0,0 +1,180 @@
+/**
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.watson.health.fhir.bulkexport;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.UUID;
+import java.util.logging.Logger;
+
+import javax.batch.api.BatchProperty;
+import javax.batch.api.chunk.AbstractItemWriter;
+import javax.inject.Inject;
+
+import com.ibm.cloud.objectstorage.ClientConfiguration;
+import com.ibm.cloud.objectstorage.SDKGlobalConfiguration;
+import com.ibm.cloud.objectstorage.auth.AWSCredentials;
+import com.ibm.cloud.objectstorage.auth.AWSStaticCredentialsProvider;
+import com.ibm.cloud.objectstorage.auth.BasicAWSCredentials;
+import com.ibm.cloud.objectstorage.client.builder.AwsClientBuilder.EndpointConfiguration;
+import com.ibm.cloud.objectstorage.oauth.BasicIBMOAuthCredentials;
+import com.ibm.cloud.objectstorage.services.s3.AmazonS3;
+import com.ibm.cloud.objectstorage.services.s3.AmazonS3ClientBuilder;
+import com.ibm.cloud.objectstorage.services.s3.model.CreateBucketRequest;
+import com.ibm.cloud.objectstorage.services.s3.model.ObjectMetadata;
+import com.ibm.cloud.objectstorage.services.s3.model.PutObjectRequest;
+
+/**
+ * Bulk export Chunk implementation - the Writer.
+ * 
+ * @author Albert Wang
+ */
+public class ChunkWriter extends AbstractItemWriter {
+    private final static Logger logger = Logger.getLogger(ChunkWriter.class.getName());
+    private AmazonS3 cosClient = null;
+    /**
+     * The IBM COS API key or S3 access key.
+     */
+    // eg "W00YiRnLW4a3fTjMB-oiB-2ySfTrFBIQQWanc--xxxx"
```

Comment:
same comments as above remove and collapse comments

Reply:
done